### PR TITLE
Add dq initialization in yt yql plugin

### PR DESCRIPTION
**Pull request by Krisha11**
### Changelog entry 
Add dq initialization in yt yql plugin

### Changelog category 
* Experimental feature

### CODE REVIEW

File: ydb/library/yql/yt/native/dq_manager.cpp [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+        .Default();
+    registrar.Parameter("scheduler", &TThis::Scheduler)
+        .Default();
+    registrar.Parameter("interconnect_settings", &TThis::ICSettings)
+        .Default();
+}
+
+//////////////////////////////////////////////////////////////////////////////
+
+TDqManager::TDqManager(const TDqManagerConfigPtr& config)
```
Comment (zlobober) [inline ydb/library/yql/yt/native/dq_manager.cpp:52]: Having such hard logic in constructor is a somewhat discouraged practice. Let's introduce another method "void Start()". Constructor must have "static" initialization and, optionally, parsing of configs, while Start() must actually have some externally visible effect, e.g. perform some network activity, expose this coordinator or start vanilla operations.

File: ydb/library/yql/yt/native/dq_manager.h [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+////////////////////////////////////////////////////////////////////////////////
+
+DECLARE_REFCOUNTED_STRUCT(TDqManagerConfig)
+DEFINE_REFCOUNTED_TYPE(TDqManagerConfig)
+
+////////////////////////////////////////////////////////////////////////////////
+
+class TDqManager
+    : public virtual TRefCounted
```
Comment (zlobober) [inline ydb/library/yql/yt/native/dq_manager.h:51]: virtual looks redundant here

File: ydb/library/yql/yt/native/plugin.cpp [hunk 3]
```diff
@@ -188,6 +206,14 @@ class TYqlPlugin
...
                 }
                 FuncRegistry_->LoadUdfs(path, emptyRemappings, 0);
+                if (DqManagerConfig_) {
+                    DqManagerConfig_->UdfsWithMd5.emplace(path, MD5::File(path));
+                }
+            }
+
+            if (DqManagerConfig_) {
+                DqManagerConfig_->FileStorage = FileStorage_;
+                DqManager_ =  New<TDqManager>(DqManagerConfig_);
```
Comment (zlobober) [inline ydb/library/yql/yt/native/plugin.cpp:216]: Extra space

**Review by zlobober — APPROVED** [review]
LGTM from my side

**Review by resetius — APPROVED** [review]

**Pull request merged**